portfolio/src/main/java/com/google/sps/servlets/Comment.java

```diff
@@ -1,20 +1,18 @@
 package com.google.sps.servlets;
 public class Comment {
 
-
-    String user = "";
-	  String comment = "";
+    String userEmail = "";
```

Comment:
I like how you were careful to make your variable names specific! (meaning userEmail instead of just user)